test/integration/default/serverspec/agent_spec.rb

```diff
@@ -20,7 +20,7 @@
   it { should contain 'Initialization complete' }
 end
 
-%w(cpu df disk swap memory load).each do |plugin|
+%w(cpu disk swap memory load).each do |plugin|
```

Comment:
In https://github.com/AerisCloud/ansible-librato/commit/88612d1a1aa5868fe270fd4b88a808464325b84f, I had moved `df` from being a default to `librato_enabled_plugins` to allow `df` to be configurable. So testing `df` was moved from here to `test/integration/default/default.yml`. But I forgot to add a `df_spec.rb` file to run a test against `df`. I'll add that in.